### ISSUE TITLE
geometric aware viewpoint classification loss function

### ISSUE THREAD

**Issue by betterze** [haotian-wang/viewpoint-estimation#1]
Dear haotian-wang,

Thank you for your nice paper and code. I really like it.

I get a little confused about the Lcls (equation 2 in page 5). Could you tell me how you represent y? It is a vector that represent azimuth or elevation, that everything is zero except the ground truth position is 1, right? 

And what is the exact function of W(v,y)? In the implement of Lcls, it just write: 

loss_class = F.cross_entropy(out_cad, labels_cad)

How do you add the W into the Lcls loss?

Thank you very much for your help.

Best Wishes,

Alex
